src/test/java/com/vdurmont/emoji/EmojiParserTest.java

```diff
@@ -30,6 +30,22 @@ public void parseToAliases_replaces_the_emojis_by_one_of_their_aliases() {
   }
 
   @Test
+  public void replaceAllEmojis_replace_the_emojis_by_character() throws Exception {
+    // GIVEN
+    String str = "An 😀awesome 😃string with a few 😉emojis!";
+
+    // WHEN
+    String result = EmojiParser.replaceAllEmojis(str, ":)");
+
+    // THEN
+    assertEquals(
```

Comment:
The indentation is a bit weird, we use double spaces everywhere else.

Reply:
Try it's works